internal/pkg/saelastic/utils.go

```diff
@@ -13,6 +14,39 @@ type EventNotification struct {
 	StartsAt    string
 }
 
+//SanitizeSummary ... summay is description and it adds double quote which breaks the json
+func SanitizeSummary(text string) string {
```

Comment:
I'm not sure I love calling out a specific parameter in the JSON and filtering across it. I have found a particular situation that results in a failed parse, but there are likely other situations that we haven't identified yet that would have a similar situation.

Reply:
@leifmadsen , unfortunately, it has to be string manipulation.  check this playground to see how single escape is also invalid in JSON when it is in name field . Collectd should treat it has a valid JSON string. I will keep looking if there are a better way to fix this. I think the solution I have now should work for all the event-based plugins.
https://play.golang.org/p/PZO3Vy7o6KZ